.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!-- Keep the progress checklist.  -->
 ## Progress
```

Comment:
My thought was that the terra-core team would maintain the progress checklist. I'm thinking now thought that we maintain this via moving issues around on our project board / adding labels and not burden non-core contributors with this.